catalogs/catalog-jdbc-mysql/src/test/java/org/apache/gravitino/catalog/mysql/integration/test/CatalogMysqlIT.java

```diff
@@ -196,10 +197,10 @@ private void createCatalog() throws SQLException {
     Catalog loadCatalog = metalake.loadCatalog(catalogName);
     Assertions.assertEquals(createdCatalog, loadCatalog);
 
-    catalog = loadCatalog;
+    return loadCatalog;
   }
 
-  private void createSchema() {
+  private void createSchema(Catalog catalog, String schemaName) {
```

Comment:
I don't think you need to pass Catalog here.
`createSchema()` is a private function, and it can access `catalog` anyway.

Reply:
because the method also used to create a schema under a specific catalog, see the 267 line in `testDropCatalog()`